android/src/app/organicmaps/widget/placepage/PlacePageController.java

```diff
@@ -96,6 +97,8 @@ public void onStateChanged(@NonNull View bottomSheet, int newState)
     @Override
     public void onSlide(@NonNull View bottomSheet, float slideOffset)
     {
+      if (mMwmActivity.isFinishing() || mMwmActivity.isDestroyed())
```

Comment:
Let's log this case as an error (does logging error crashes/asserts now in debug? C++ code asserts.)